libraries/RC_Channel/RC_Channel.h

```diff
@@ -219,16 +219,17 @@ class RC_Channel {
         // also, if you add an option >255, you will need to fix duplicate_options_exist
 
         // options 150-199 continue user rc switch options
-        CRUISE =             150,  ///CRUISE mode
+        CRUISE =             150,  // CRUISE mode
         TURTLE =             151,  // Turtle mode - flip over after crash
         SIMPLE_HEADING_RESET = 152, // reset simple mode refernce heading to current
         ARMDISARM =          153, // arm or disarm vehicle
         ARMDISARM_AIRMODE =  154, // arm or disarm vehicle enabling airmode
         TRIM_TO_CURRENT_SERVO_RC = 155, // trim to current servo and RC
         TORQEEDO_CLEAR_ERR = 156, // clear torqeedo error
-        EMERGENCY_LANDING_EN = 157, //Force long FS action to FBWA for landing out of range
+        EMERGENCY_LANDING_EN = 157, // Force long FS action to FBWA for landing out of range
+        HEADING_HOLD =       158,  // select heading hold or rate mode for yaw controller
```

Comment:
This probably needs to be added to the parameter description for plane as well..

Reply:
thanks, I had forgotten about that